tests/console/ansible.pm

```diff
@@ -100,12 +114,20 @@ sub run {
     die 'network-engine should be installed!' unless $galaxy_installed =~ m/ansible-network\.network-engine/;
 
     # 4. Ansible playbook testing
+    my $skip_tags;
+    $skip_tags = '--skip-tags zypper' if is_transactional;
 
     # Check that community.general.zypper module is available
     assert_script_run 'ansible-doc -l community.general | grep zypper';
 
+    # Check the version of ansible-community from where we use the zypper module
+    script_run 'ansible-community --version';
+
+    # Check that a transactional system has the necessary files
+    script_run 'ls -la /var/lib/misc/' if is_transactional;
```

Comment:
It would be more straightforward to check existence of necessary files. Not sure, what happens when `{health-check,transactional-update}.state` are missing? Would there be some other obvious error later on?

Reply:
I will remove this for now as it makes no sense because I don't test the zypper ansible module on transactional systems right now - I would like to do so later in separate pull request.